Tests/scripts/validate_files_structure.py

```diff
@@ -97,6 +100,10 @@
 DIRS = [INTEGRATIONS_DIR, SCRIPTS_DIR, PLAYBOOKS_DIR, REPORTS_DIR, DASHBOARDS_DIR, WIDGETS_DIR, INCIDENT_FIELDS_DIR,
         LAYOUTS_DIR, CLASSIFIERS_DIR, MISC_DIR]
 
+# secrets white list
+ENTROPY_THRESHOLD = 3.4
```

Comment:
perhaps we should this to be configurable with `3.4` being the default?

Reply:
Could be, configureable from where tho. Right now editing this var will immediately take affect.